gridstatus/ercot_api/ercot_api.py

```diff
@@ -1191,6 +1196,67 @@ def get_60_day_dam_disclosure(
         # Take the list of dictionaries and concat the dataframes for each key
         return {key: pd.concat([d[key] for d in df_list]) for key in df_list[0].keys()}
 
+    @support_date_range(frequency=None)
+    def get_60_day_sced_disclosure(
+        self,
+        date: str | pd.Timestamp,
+        end: str | pd.Timestamp = None,
+        verbose: bool = False,
+    ) -> Dict[str, pd.DataFrame]:
+        """
+        Get the 60-day SCED disclosure reports from ERCOT.
+
+        Args:
+            date (datetime-like): Start date for the query
+            end (datetime-like, optional): End date for the query.
+                Defaults to date + 1 day
+            verbose (bool, optional): Whether to print progress messages. Defaults to
+                False
+
+        Returns:
+            dict: Dictionary containing dataframes as values and keys:
+                - "sced_gen_resource"
+                - "sced_load_resource"
+                - "sced_smne"
+
+        NOTE: because data is delayed by 60 days, requesting data in the past 60 days
+        will return no data.
+        """
+        # Reports are delayed by 60 days
+        date = date + pd.DateOffset(days=60)
+
+        # End is required so set a default end date
+        if end:
+            end = end + pd.DateOffset(days=60)
+        else:
+            end = date + pd.DateOffset(days=1)
+
+        df_list = []
+
+        # Get data once since both endpoints return the same zipfile
+        data_bytes = self.get_historical_data(
+            endpoint=SCED_60_DAY_SMNE_ENDPOINT,
```

Comment:
am i understanding correctly that `SCED_60_DAY_SMNE_ENDPOINT` returns a zip file with all of the datasets we care about, not just the `sced_smne`?

Reply:
Yes. If you hit the endpoint for any of the files within the zipfile, it will return the same zipfile.